README.md

```diff
@@ -114,6 +114,14 @@ In this scenario you will incorporate WebChat's multiple JavaScript files into y
 5. Create an instance of `DirectLine` using your Direct Line Secret or Token
 6. Call the `Chat` React component with your DirectLine instance, user and bot identities, and other properties as demonstrated in [samples/index.html](https://github.com/Microsoft/BotFramework-WebChat/blob/master/samples/index.html). 
 
+#### Using the Backchannel
+NOTE: The provided backchannel sample requires a bot which can send and receive specific event messages. Follow the instructions [here](https://github.com/ryanvolum/backChannelBot) to deploy such a bot. 
+
+Backchannel is a way to send event messages (which are invisible to the actual webchat) between the page that hosts webchat and your bot web service. The backchannel sample provided in this project listens for events of activity name "changeBackground" and sends events of activity name "buttonClicked". This highlights the ability for a bot to communicate with a page that embeds the bot through WebChat. In other words, our bot can:
```

Comment:
1. "... event messages... " -> "... activities of type 'event'... "
2. "invisible to" -> "ignored by"
3. "events of activity name ... " -> "events with name ..."